### PR TITLE
Google Custom Search starter-kit

### PR DESCRIPTION
**Pull request by sne3091**
Starter kit for search using Google Custom Search JSON API

### CODE REVIEW

File: integrations/extensions/starter-kits/google-search/README.md
Comment (jwm4) [inline integrations/extensions/starter-kits/google-search/README.md:7]: Should be endpoints, not endpoint since there are two.

File: integrations/extensions/starter-kits/google-search/README.md
Comment (jwm4) [inline integrations/extensions/starter-kits/google-search/README.md:17]: Also the "Custom Search Site Restricted" API is only available for a programmable search that is restricted to only searching specific sites (10 or fewer).  In contrast, "Custom Search" can do full web search as I understand it.  This is mentioned in the same link but the wording is confusing.

**Review by amiddavid — APPROVED** [review]
from a structure perspective this looks complete. I trust @jwm4 to test it actually works

**Review by jwm4 — APPROVED** [review]
My review comments are all addressed, and I tested it out and verified that it works, so I think this will be ready to merge as soon as Arnesh provides legal approval.

**Comment by sne3091** [comment]
Converted back to draft, awaiting legal approval

**Comment by sne3091** [comment]
Closing in favor of https://github.com/watson-developer-cloud/assistant-toolkit/pull/54

**Pull request closed without merging**
